### PR TITLE
My First Pull

### PR DESCRIPTION
**Pull request by Lea-Sh**
Hi! I try to do anything but I don't understand exactly what I do... Good night, sweet dreams... 

(She wants to sleep right now @Gadd) Таки I do!

### CODE REVIEW

File: MultiThreadSol.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+ */
+public class MultiThreadSol implements Runnable {
+    private long start, fin;
+    MultiThreadSol(long start, long fin){
+        this.start= start;
+        this.fin=fin;
+    }
+    @Override
+    public void run() {
+        OneThreadSolution sol= new OneThreadSolution();
```
Comment (pasby) [inline MultiThreadSol.java:12]: please use code autoformating ) need spase before '='

File: OneThreadSolution.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+/**
+ * Created by user on 02.04.2016.
+ */
+public class OneThreadSolution {
+
+    private boolean hasDifferentDigits(long l) {
+        int digit;
+        Set<Integer> digits = new HashSet<>();
+        digits.add(0);
+        while (l > 0) {
```
Comment (pasby) [inline OneThreadSolution.java:13]: you can make method here and reuse it further

File: Solve.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    public static void main(String[] args) {
+        System.out.print("One Thread: ");
+        OneThreadSolution sol = new OneThreadSolution();
+
+        sol.solve(123456789,987654321);
+
+
+        System.out.print("Five Threads: ");
+       Thread[] sols= new Thread[5];
+        sols[0]= new Thread(new MultiThreadSol(123_456_789,298_765_431));
```
Comment (pasby) [inline Solve.java:14]: no need to store threads.. you can start them rightaway

File: Solve.java [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+        sols[1]= new Thread(new MultiThreadSol(312_456_789,498_765_321));
+        sols[2]= new Thread(new MultiThreadSol(512_346_789,698_754_321));
+        sols[3]= new Thread(new MultiThreadSol(712_345_689,897_654_321));
+        sols[4]= new Thread(new MultiThreadSol(912_345_678,987_654_321));
+
+        for (int i = 0; i < 5; i++) {
+            sols[i].start();
+        }
+
+
```
Comment (pasby) [inline Solve.java:24]: please, use code autoformat to make code more beautiful )

File: OneThreadSolution.java (hunk 2 above)
Comment (Lea-Sh) [inline OneThreadSolution.java:13]: If I make a method here, then after reuse I'll need additional loops. Is it really better than this variant?

**Pull request closed without merging**